scripts/certs.sh

```diff
@@ -94,77 +103,109 @@ starter() {
   info "Initialize environment..."
 
   local RES_FILE=$(mktemp /tmp/init_env.XXXX)
-  local STATUS_CODE=$(k8s_api_call "GET" "/apis/extensions/v1beta1/namespaces/${NAMESPACE}/ingresses" 2>${RES_FILE})
 
-  if [ "${STATUS_CODE}" = "200" ]; then
+  # try get namespaces (cluster mode)
+  status_code=$(k8s_api_call "GET" /api/v1/namespaces 2>"${RES_FILE}")
```

Comment:
Deployment cluster scoped will failed. Please update your code to manage this case.
Use `local` keyword for variables when needed.

Reply:
> Deployment cluster scoped will failed. Please update your code to manage this case.
> Use `local` keyword for variables when needed.

What error did you see?

Reply:
> Use `local` keyword for variables when needed.

I don't think `sh` supports `local`.

Reply:
> What error did you see?
Cluster scoped deployment will failed due to a call requesting all namespaces without having a clusterrole. Just try it.

> I don't think sh supports local.
ash is the system shell used on busybox and local keyword are supported.

Reply:
> Cluster scoped deployment will failed due to a call requesting all namespaces without having a clusterrole. Just try it.

If you set cluster to `true` you will have a ClusterRole. If you set cluster to `false`, the request will fail. This is expected and will not cause problems. I've tested on my cluster. Maybe you need a fresh install? See `certs/templates/rolebinding.yaml`.

Reply:
> ash is the system shell used on busybox and local keyword are supported.

I'll look into this.